clouddriver-openstack/src/test/groovy/com/netflix/spinnaker/clouddriver/openstack/client/OpenstackClientProviderSpec.groovy

```diff
@@ -1378,6 +1420,40 @@ class OpenstackClientProviderSpec extends Specification {
     noExceptionThrown()
   }
 
+  def "get instance ids for stack succeeds" () {
+    setup:
+    HeatService heat = Mock()
+    ResourcesService resourcesService = Mock()
+
+    when:
+    provider.getInstanceIdsForStack(region, "mystack")
+
+    then:
+    1 * mockClient.heat() >> heat
+    1 * heat.resources() >> resourcesService
+    1 * resourcesService.list("mystack")
+    noExceptionThrown()
```

Comment:
Can you ensure the filtering for just `OS:Nova:Server` resources is happening correctly